csharp/regenerate_paket2bazel_files.sh

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
If this is meant to be checked in, we need a better name!

Reply:
I'd like to check it in not to copy-paste the commands from a doc, but to run one script without excessive thoughts. Any ideas regarding the name?

Reply:
Moved every paket2bazel file into the new nuget/ directory. Renamed the file to "update.sh".

Reply:
https://github.com/vaticle/typedb-driver/commit/204555cb2ff2a2d0b841751a2b56a2219fc66f30